### DISCUSSION TITLE
Backspace key not working in telnet OLT

### DISCUSSION THREAD

**Discussion by spandanpradhan** [Eugeny/tabby#8859] [Q&A]
Hi,
I am unable to delete a word by using backspace key , while doing telnet/ssh for OLT.
But it works fine for ssh in switch and router.

Please assist.

**Comment by edwinx333** [comment]
backspace is not working for me in my Telnet profiles either.

**Reply by Eugeny** [reply]
You can choose the control character that the backspace key is sending in the profile settings.
